src/points/types.ts

```diff
@@ -84,6 +85,7 @@ interface PointsEventCreateWallet {
 interface PointsEventSwap {
   activityId: 'swap'
   transactionHash: string
+  network: Network
```

Comment:
What do you think of using `NetworkId` to remove any ambiguity?

Reply:
yes fair, i updated to use networkId. i thought i was saving some computation by using the network but i was mistaken (reading the wallet code i thought that we need only the network to create the viem client, but we've a couple of layers of mappings in between!)